tools/validateJSON.js

```diff
@@ -137,7 +138,8 @@ function cleanValidationErrorProperties(tv4ValidationError) {
 function addMissingSchemas(missingUris, schemaFolderPath) {
     while (missingUris && missingUris.length > 0) {
         var missingUri = missingUris.pop();
-        if (missingUri && missingUri.length > 0) {
+       console.log(missingUri);
+        if (missingUri && missingUri.length > 0 && missingUri != 'https://schema.management.azure.com/schemas/2017-10-01/microsoft.insights.json' && missingUri != 'https://schema.management.azure.com/schemas/2016-03-01/microsoft.insights.json') {
```

Comment:
do you need this change?

Reply:
same, committed by mistake, reverted.

This change is required on Linux to make tests work